cpp/src/strings/regex/regex.cuh

```diff
@@ -42,10 +42,10 @@ class reprog;
 using match_pair   = thrust::pair<cudf::size_type, cudf::size_type>;
 using match_result = thrust::optional<match_pair>;
 
-constexpr int32_t RX_STACK_SMALL  = 112;    ///< fastest stack size
-constexpr int32_t RX_STACK_MEDIUM = 1104;   ///< faster stack size
-constexpr int32_t RX_STACK_LARGE  = 10128;  ///< fast stack size
-constexpr int32_t RX_STACK_ANY    = 8;      ///< slowest: uses global memory
+constexpr int32_t RX_STACK_SMALL  = 112;   ///< fastest stack size
+constexpr int32_t RX_STACK_MEDIUM = 1104;  ///< faster stack size
+constexpr int32_t RX_STACK_LARGE  = 2560;  ///< fast stack size
```

Comment:
This change will conflict with #10573, right? Unless git is smart enough to recognize that it's an identical change, which in my experience works sometimes but not always.

Reply:
(This doesn't really matter, just a heads up in case there are other conflicts and you have a preferred merge order)

Reply:
Yes. There will be many like these to resolve. I made the choice to separate all the changes into separate PRs. I do not fear the conflict resolutions.

Reply:
> I do not fear the conflict resolutions.

@davidwendt is a man of true courage